cmd/agent_autopilot.go

```diff
@@ -19,15 +21,30 @@ var agentAutopilotCmd = &cobra.Command{
 	Short: "Background service that automatically repays FIL to pools",
 	Long:  `Background service that automatically repays FIL to pools.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		defer func() {
+			if r := recover(); r != nil {
+				fmt.Println("stacktrace from panic: \n" + string(debug.Stack()))
+			}
+		}()
+
 		defer journal.Close()
 
+		if cmd.Flag("logfile") != nil && cmd.Flag("logfile").Changed {
```

Comment:
Is this a new user defined "thing"? or is this separately the journal?

Reply:
new just for this command, defaults empty.